pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -22,11 +22,12 @@ import (
 )
 
 const (
-	agentName        = "volumemgr"
-	runDirname       = "/var/run/" + agentName
-	ciDirname        = runDirname + "/cloudinit" // For cloud-init volumes XXX change?
-	rwImgDirname     = types.RWImgDirname        // We store volumes here
-	roContImgDirname = types.ROContImgDirname    // We store container bootable disks here
+	agentName              = "volumemgr"
+	runDirname             = "/var/run/" + agentName
+	ciDirname              = runDirname + "/cloudinit"    // For cloud-init volumes XXX change?
+	rwImgDirname           = types.RWImgDirname           // We store volumes here
+	roContImgDirname       = types.ROContImgDirname       // We store container bootable disks here
+	volumeEncryptedDirName = types.VolumeEncryptedDirName // We store container bootable disks here
```

Comment:
Shouldn't the comment be saying "VM and OCI volumes here"?

Reply:
Corrected.